internal/ingress/controller/controller.go

```diff
@@ -591,11 +591,13 @@ func (n *NGINXController) getBackendServers(ingresses []*ingress.Ingress) ([]*in
 									location.Path, server.Hostname, location.DefaultBackend.Namespace, location.DefaultBackend.Name)
```

Comment:
similar to https://github.com/kubernetes/ingress-nginx/pull/3730/files#r256678420, this log message can be lie, maybe move it to https://github.com/kubernetes/ingress-nginx/pull/3730/files#diff-86ac9ff9d75a0c5005c116e766a6127dR598?

Reply:
done as well, good catch - I had missed that entirely